docs/Contact_Candidate.html

```diff
@@ -15,4 +15,4 @@
 </pre></div> </div> </div> </div> </div> </div> <div class=accordion id=accordionoccupation> <div class=card> <div class=card-header id=headingoccupation> <h2 class=mb-0> <button class="btn btn-link property-name-button" type=button data-toggle=collapse data-target=#occupation aria-expanded aria-controls=occupation onclick="setAnchor('#occupation')"><span class=property-name>occupation</span></button> </h2> </div> <div id=occupation class="collapse property-definition-div" aria-labelledby=headingoccupation data-parent=#accordionoccupation> <div class="card-body pl-5"> <div class=breadcrumbs>root <svg width=1em height=1em viewbox="0 0 16 16" class="bi bi-arrow-right-short" fill=currentColor xmlns=http://www.w3.org/2000/svg> <path fill-rule=evenodd d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"/> </svg> <a href=#occupation onclick="anchorLink('occupation')">occupation</a></div><span class="badge badge-dark value-type">Type: string or null</span><br> <span class=pattern-value id=occupation_pattern>Must match regular expression: <code>^[ -~]{0,38}$</code></span> <br> <div class="badge badge-secondary">Example:</div> <br><div id=occupation_ex1 class="jumbotron examples"><div class=highlight><pre><span></span><span class=s2>&quot;QC Inspector&quot;</span><span class=w></span>
```

Comment:
Because it makes it hard to find what changed otherwise, I'd recommend only committing the html for the schemas that changed.  The other .htmls are the same with timestamp differences that are inconsequential

Reply:
I have been doing blanket docs/*.html changes using the bin/generate-schema-docs.sh script so that the Security Hotspot issue always reported by SonarCloud is corrected. I think it okay to ignore the files in ../docs in CR, except for the index.html file, since they are automatically generated but do agree it muddies up the diff inspection.

I would like to require the docs/*.html files be generated exclusively by the bin/generate-schema-docs.sh script so let's plan on updating the script or making an additional script that generates the *.html file for a single schema.

Reply:
Actually, the best solution for this is to have the docs content automatically generated at commit ... there may be a ticket somewhere for that...

Reply:
I did notice that hotspot.  So I ran `bin/generate-schema-docs.sh` and just left the other htmls out of the commit.  You're saying that the hotspot is caused by not including the other files?

Reply:
The hotspot is caused by our open source generate-schema-doc tool when it creates the *.html file. It has a fontawsome <script> reference that triggers the issue in SonarCloud. The bin/generate-schema-docs.sh script has a sed call that removes the offending snippet on the fly. We just need to adapt the script to also handle a single schema in addition to looping through the whole directory.